proto/osmosis/protorev/v1beta1/protorev.proto

```diff
@@ -87,6 +87,8 @@ message PoolWeights {
   // The weight of a concentrated pool
   uint64 concentrated_weight = 3
       [ (gogoproto.moretags) = "yaml:\"concentrated_weight\"" ];
+  // The weight of a cosmwasm pool
+  uint64 cosmwasm_weight = 4 [ (gogoproto.moretags) = "yaml:\"cw_weight\"" ];
```

Comment:
We may need to make this dynamic based on address in the future?

Reply:
Yea totally agree. Ill add some tasks to the ProtoRev v17 issue tracker.